OptimoveSDK/Sources/Classes/PreferenceCenter/PreferenceCenterModels.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+struct PreferenceCenterConfig: Codable {
+    var region: Region
+    var tenantId: Int
+    var brandGroupId: String
+}
+
+public class Preferences: NSObject, Codable {
```

Comment:
Consider using struct first. 
In this case, with exposing struct to objc, we need to wrap up the Preference struct with PrefernceObjc class. So, we will have doubled API. It will help us to use only modern lang features internally in sdk. 

The sdk should have some examples of it. I will find it